### ISSUE TITLE
The default_theme in config.php file is set to 'Sugar' after CLI upgrade

### ISSUE THREAD

**Issue by likhobory** [salesagility/SuiteCRM#7169]
#### Issue

If we performing an upgrade from the CLI as described in PR #6792, the default_theme in config.php file is set to 'Sugar'.

https://github.com/salesagility/SuiteCRM/blob/v7.11.3/modules/UpgradeWizard/silentUpgrade_step1.php#L835

#### Actual Behavior
```
logThis('Set default_theme to Sugar', $path);
            $sugar_config['default_theme'] = 'Sugar';
```

#### Steps to Reproduce

Run SilentUpgrade and check the result in the log and config.php files.

#### Your Environment

* SuiteCRM Version used: 7.11.3

**Comment by likhobory** [comment]
I also propose to update the Silent Upgrade Usage message, now it displays this:

```
Usage: php.exe -f silentUpgrade.php [upgradeZipFile] [logFile] [pathToSugarInstance] [admin-user]

On Command Prompt Change directory to where silentUpgrade.php resides. Then type path to
php.exe followed by -f silentUpgrade.php and the arguments.

Example:
    [path-to-PHP/]php.exe -f silentUpgrade.php [path-to-upgrade-package/]SugarEnt-Upgrade-5.2.0-to-5.5.0.zip [path-to-log-file/]silentupgrade.log  [path-to-sugar-instance/] admin

Arguments:
    upgradeZipFile                       : Upgrade package file.
    logFile                              : Silent Upgarde log file.
    pathToSugarInstance                  : Sugar Instance instance being upgraded.
    admin-user                           : admin user performing the upgrade
```

**Comment by likhobory** [comment]
Fixed in PR #7562.
